src/docs/sdk/overview.mdx

```diff
@@ -229,6 +230,24 @@ The following additional headers are permitted as per CORS policy:
 - `content-encoding`
 - `transfer-encoding`
 
+### User Agent
+
+All SDKs are expected to report their name and version via the `user-agent` header. 
+The following format should be used (unless required or recommended differently by the platform, 
+e.g. for browser SDKs):
+
+`{sdk-name}/{sdk-version}`
```

Comment:
Can we list some examples here? IMO it would make sense if the user agent is always prefixed by `sentry-`, not sure if that's included in the `{sdk-name}`.

Reply:
most SDKs I checked use `sentry.{name}`, ruby uses `sentry-ruby`. if we define a requirement for `sentry-` we'd need to adapt a bunch of SDKs - I'm gonna add the examples I found

Reply:
👍 As long as they have `sentry` in the name I think that's fine. I was worried about the user agent showing up as `python/1.2.3`, which would make it hard to attribute to sentry.